docs/reference/query_proxy.rst

```diff
@@ -10,6 +10,7 @@ The ``ProxyQuery`` object is used to add missing features from the original `Doc
 * ``execute`` method - no need to call the ``getQuery()`` method,
 * add sort by and sort order options,
 * add preselect id query on left join query, so a limit query will be only applied on the left statement and not on the full select statement. This simulates the original Doctrine 1 behavior.
+* For disable distinct call method ``setDistinct(false)``. By default distinct is enable.
```

Comment:
This requires way more explanation IMO. I don't fully understand this PR yet, but here is what I think could be here, tell me if/where I'm wrong, and please fill in the blanks.

By default Sonata will use the `DISTINCT` SQL keyword, because <insert good reason here>. This is not required for queries that <insert description of queries that do not need DISTINCT (or maybe it's necessary on some RDBMS' and not others? I don't know)>. Sonata cannot detect whether or not you need `DISTINCT`, but lets you remove that keyword in case it causes performance issues. To do so, simply call ``setDistinct(false)``